Makefile

```diff
@@ -19,6 +19,8 @@ DEP_CHECK_OUT_FILE_NAME := $(PREFIX)__$(ORG_NAME)__$(REPO_NAME)__$(BRANCH_NAME)_
 
 DOCKER_COMPOSE_OUT_FILE_NAME := ./infrastructure/output/$(PREFIX)__docker__compose__output__$(TIMESTAMP).txt
 
+NOVE_VERSION := v18.16.0
```

Comment:
Minor typo, this should be NODE_VERSION.

Reply:
Thanks @Harrisman05 will do 👍